vita3k/gui/src/gui.cpp

```diff
@@ -389,16 +388,14 @@ void init_app_background(GuiState &gui, EmuEnvState &emuenv, const std::string &
     else
         vfs::read_app_file(buffer, emuenv.pref_path.wstring(), app_path, "sce_sys/pic0.png");
 
-    const auto title = (APP_INDEX != gui.app_selector.sys_apps.end()) && (APP_INDEX != gui.app_selector.user_apps.end()) ? APP_INDEX->title : app_path;
```

Comment:
I no see any problèm here ?
Is just fine 🤔, same code is used also in other place

Reply:
It crash on debug build at least.

Reply:
Is strange but ok 🤔